### PR TITLE
STP-3180: Removing 1.5 specific heading in BOS topic

### PR DESCRIPTION
**Pull request by nrockershousen**
## Summary and Scope

Removing the 1.5 specific heading and doing some minor editing throughout the file.

## Issues and Related PRs

* Resolves [STP-3180](https://jira-pro.its.hpecorp.net:8443/browse/STP-3180)
* Merge with https://github.com/Cray-HPE/docs-csm/pull/1686

## Testing

N/A

### Tested on:

N/A

### Test description:

N/A

## Risks and Mitigations

None.

## Pull Request Checklist

- [ ] Version number(s) incremented, if applicable
- [ ] Copyrights updated
- [ ] License file intact
- [ ] Target branch correct
- [ ] CHANGELOG.md updated
- [ ] Testing is appropriate and complete, if applicable
- [ ] [HPC Product Announcement](https://cray.slack.com/archives/C026TVCSXLH) prepared, if applicable

### CODE REVIEW

File: operations/boot_orchestration/Sessions.md [hunk 1]
```diff
@@ -1,34 +1,43 @@
...
+BOA will then provide a command in the output of the BOA log that can be used to retry the action.
+This behavior impacts all BOS operations.
 
-A good example would be if there is a 6,000 node system and 3 nodes fail to power off during a BOS operation. BOA will continue and attempt to re-provision the remaining 5,997 nodes. After the command is finished, it will provide output information about what the administrator needs to do to retry the last 3 nodes that failed.
+For example, if there is a 6,000 node system and 3 nodes fail to power off during a BOS operation.
+BOA will continue and attempt to re-provision the remaining 5,997 nodes.
+After the command is finished, it will provide output information about what the administrator needs to do to retry the last 3 nodes that failed.
 
-### Current BOS Session Limitations
+## Current BOS Session Limitations
```
Comment (mharding-hpe) [inline operations/boot_orchestration/Sessions.md:34]:
```suggestion
## Current BOS session limitations
```

**Review by mharding-hpe — CHANGES_REQUESTED** [review]

File: operations/boot_orchestration/Sessions.md [hunk 2]
```diff
@@ -1,34 +1,43 @@
...
+The endpoint can also display the details of a given session when the specific session ID is provided as a parameter. Sessions can also be deleted through the API.
 
-A session requires two parameters, a session template ID and an operation to perform on that template. The BOS API's `session` endpoint can display a list of all of the sessions that have been created, including previous and currently running sessions. The endpoint can also display the details of a given session when the specific session ID is provided as a parameter. Sessions can also be deleted through the API.
+BOS supports a RESTful API. This API can be interacted with directly using tools like cURL.
+It can also be interacted with through the Cray Command Line Interface \(CLI\). See [Manage a BOS Session](Manage_a_BOS_Session.md) for more information.
 
-BOS supports a RESTful API. This API can be interacted with directly using tools like cURL. It can also be interacted with through the Cray Command Line Interface \(CLI\). See [Manage a BOS Session](Manage_a_BOS_Session.md) for more information.
+## BOA Functionality
```
[the remark this answers is not in the document]
Comment (nrockershousen) [reply operations/boot_orchestration/Sessions.md:21]: Just a note on the style guide as our team has been discussing this recently. The Knowledge Management team (HPE version of Cray tech pubs) has a legacy HPE style guide and the Cray team has a legacy style guide intended for technical publications. The "HPE writing style guide" was created outside of HPE KM and Cray tech pubs. There are discrepancies between the three, and our team is trying to standardize some of the bigger consistency issues, such as headings. 

As long as we are being consistent within this publication this change is good. Just wanted to note this as there may be a different style in the future.

File: operations/boot_orchestration/Sessions.md [hunk 3]
```diff
@@ -1,34 +1,43 @@
...
+If there are transient failures, BOA will exit and Kubernetes will reschedule it so that it can re-execute its session.
 
-In release 1.5, BOA moves nodes towards the requested state, but if a node fails during any of the immediate steps, it takes note of it. BOA will then provide a command in the output of the BOA log that can be used to retry the action. This behavior impacts all BOS operations.
+BOA moves nodes towards the requested state, but if a node fails during any of the immediate steps, it takes note of it.
+BOA will then provide a command in the output of the BOA log that can be used to retry the action.
+This behavior impacts all BOS operations.
 
-A good example would be if there is a 6,000 node system and 3 nodes fail to power off during a BOS operation. BOA will continue and attempt to re-provision the remaining 5,997 nodes. After the command is finished, it will provide output information about what the administrator needs to do to retry the last 3 nodes that failed.
+For example, if there is a 6,000 node system and 3 nodes fail to power off during a BOS operation.
```
[the remark this answers is not in the document]
Comment (nrockershousen) [reply operations/boot_orchestration/Sessions.md:30]: This should stay as 3. If one number in a category of numbers requires a numeral, the other numbers should be numerals for consistency.

[the remark this answers is not in the document]
Comment (mharding-hpe) [reply operations/boot_orchestration/Sessions.md:30]: Good to know!

**Review by mharding-hpe — APPROVED** [review]

**Review by jsollom-hpe — APPROVED** [review]

**Pull request merged**